services/prompter.js

```diff
@@ -112,6 +112,7 @@ async function Prompter(program, requests) {
     }
   }
 
+
```

Comment:
[Cleaning] What is this new line for? (Maybe the linter could prevent more than 1 newline in a raw)

Reply:
> Maybe the linter could prevent more than 1 newline in a raw

Yes: https://eslint.org/docs/rules/no-multiple-empty-lines I will create another PR for that.

> What is this new line for?

:) Useless (maybe a merge glitch). Fixed now.